src/Ops.Web/Areas/Admin/Views/Features/FeatureDetails.cshtml

```diff
@@ -0,0 +1,150 @@
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <a class="btn btn-secondary" id="return-home" asp-action="@nameof(FeaturesController.Index)">Return to Features</a>
+                    @if (Model.Feature.IsNewFeature)
+                    {
+                        <text>
+                            <button type="submit" id="submit" value="Submit" class="btn btn-primary btn-spinner">
+                                <span class="buttonText">Add Feature</span>
+                                <span class="fa fa-spinner fa-pulse fa-lg fa-fw d-none"></span>
+                            </button>
+                        </text>
+                    }
+                    else
+                    {
+                        <text>
+                            <button type="submit" id="submit" value="Submit" class="btn btn-primary btn-spinner">
+                                <span class="buttonText">Update Feature</span>
+                                <span class="fa fa-spinner fa-pulse fa-lg fa-fw d-none"></span>
+                            </button>
+                        </text>
+                    }
+                </div>
+            </div>
+        </div>
+    </form>
+</div>
+@section scripts{
+    <script src="~/devscripts/fontawesome-iconpicker/main.js" asp-append-version="true"></script>
+    <script>
+    $(function () {
+        $('.icon-picker').iconpicker();
+    });
+        
+    </script>
+    <script src="~/js/md.min.js" asp-append-version="true"></script>
+    <script>
+        $("#accordion-header").addClass("alert-success");
+
+        if(!$("#Feature_Stub").val()) {
+            $('.collapse').collapse();
+            $("#accordion-header").removeClass( "alert-success" );
+            $("#accordion-header").addClass("alert-danger");
+        }
+
+        $("#popup-button").click(function () {
+            if ($("#accordion-header").hasClass("alert-success")) {
+                $("#accordion-header").removeClass( "alert-success" );
+                $("#accordion-header").addClass("alert-danger");
+            }
+            else if ($("#accordion-header").hasClass("alert-danger")) {
+                $("#accordion-header").removeClass( "alert-danger" );
+                $("#accordion-header").addClass("alert-success");
+            }
+        });
+        $("#submit").click(function () {
+            if ($("#accordion-header").hasClass("alert-danger")) {
+                $("#wmd-input").val(" ");
+                $("input#Feature_Stub").val(" ");
+            }
+        });
+        var editor = new Markdown.Editor({ allowUploads: false });
+        editor.run();
+        $(document).ready(function () {
+
+            var stubInfo = document.createElement("span");
```

Comment:
Why is this being built when the document is ready? Can this tag be present in the HTML rather than built and appended with JavaScript?

Reply:
Its an issue with the taghelper. Dan said he could add it if needed